tests/components/color_picker.test.ts

```diff
@@ -120,7 +122,7 @@ describe("Color Picker buttons", () => {
     await simulateClick(".o-color-picker-toggler");
     setInputValueAndTrigger(".o-custom-input-preview input", "this is not a color", "input");
     await nextTick();
-    expect(fixture.querySelector(".o-wrong-color")).toBeNull();
+    expect(fixture.querySelector(".o-wrong-color")).not.toBeNull();
     await simulateClick(".o-add-button");
     expect(fixture.querySelector(".o-wrong-color")).not.toBeNull();
```

Comment:
I'm not sure the click and the "add button" and the testing part after is still needed now :)